src/icon-api.js

```diff
@@ -184,16 +174,51 @@ export function getBestVariant(variants, opts) {
 
     // Both variants match desired size, use pixel density to determine preferred match
     if (aSizeDiff === 0 && bSizeDiff === 0) {
-      // Pixel density doesn't matter for svg but prefer lower for consistent results
+      // What to do if asking for an svg and both (a and b) are svgs and have the same size, what criteria do we use?
+      // For now check a and b for mimeType, the only weird case being both svgs
       if (opts.mimeType === 'image/svg+xml') {
-        return a.pixelDensity <= b.pixelDensity ? -1 : 1
+        if (a.mimeType === 'image/svg+xml' && b.mimeType === 'image/png') {
```

Comment:
this probably can be done much cleaner (and I probably got the sort code backwards...).
 I'm basically checking every combination of `mimeType` for a and b with both `opts.mimeType` being png or svg. This is because now svgs don't have a `pixelDensity`